d3-scripts/tests/__fixtures__/duplicate-property-type-inheritance-single-parent/test-2.type.d3.yaml

```diff
@@ -0,0 +1,11 @@
+type: d3-device-type-assertion
+credentialSubject:
+  # unique identifiers
+  id: e22b0268-b6ca-42f8-9317-b629e12286ef
+
+  parents:
+    - id: ba3ba36d-be70-40f0-9c52-95b560066d11 # Id of parent type to inherit from
+      # Specific properties to inherit
+      properties:
+        - manufacturer
+        - manufacturer
```

Comment:
Duplicate?

Reply:
yeah, it's on purpose, this test claim is checking an error is thrown when duplicate properties are entered :smile: